Formula/otf2.rb

```diff
@@ -0,0 +1,27 @@
+class Otf2 < Formula
+  desc "Open Trace Format 2 file handling library"
+  homepage "https://www.vi-hps.org/projects/score-p/"
+  url "https://www.vi-hps.org/cms/upload/packages/otf2/otf2-2.1.1.tar.gz"
+  sha256 "01591b42e76f396869ffc84672f4eaa90ee8ec2a8939755d9c0b5b8ecdcf47d3"
+
+  depends_on "python"
```

Comment:
```suggestion
  depends_on "gcc"
  depends_on "open-mpi"
  depends_on "python"
```

Should build for Fortran support and MPI support too. I'm not so sure about python here;

I think without the explicit python dependency it should pick up the system python and build bindings for that.